block-storage-attacher/tests/e2e/conf/prod-us-south.conf

```diff
@@ -61,8 +61,7 @@ ETCD_SET1=etcd:https://ibm_cloud_4090972d_9e5c_4aaf_8fe7_8f0c83a2b0bd:1ee6bdff17
 ETCDCREDS=ibm_cloud_4090972d_9e5c_4aaf_8fe7_8f0c83a2b0bd:1ee6bdff1743fca761efc6df24673122af15026fe08cf75c775763bc4be231cd
 PORTWORXVER=$PORTWORXVER
 IBMSECRETTYPE=ibm-kp
-CERTPATH=/etc/pwx/etcdcerts
-CAPATH=/etc/pwx/etcdcerts/ca.pem
+ECRET=px-etcd-certs
```

Comment:
Typo here. Please change and also in places where its used.

Reply:
Addressed the comment